node/cmd/guardiand/node.go

```diff
@@ -639,13 +646,25 @@ func runNode(cmd *cobra.Command, args []string) {
 		if *neonContract == "" {
 			logger.Fatal("Please specify --neonContract")
 		}
+		if *baseRPC == "" {
+			logger.Fatal("Please specify --baseRPC")
+		}
```

Comment:
Please add the following to node/cmd/guardiand/adminserver.go::DumpRPCs() (line 779)
```
rpcMap["baseRPC"] = *baseRPC

Reply:
Done.